tests/mobly/base_test_test.py

```diff
@@ -20,6 +20,10 @@
 from mobly import config_parser
 from mobly import signals
 
+from tests.mobly import records_test
```

Comment:
Not allowed by whom?
It seems to run fine both by pytest and by direct trigger.